manifest.php

```diff
@@ -62,7 +62,7 @@
     'label' => 'TAO Base',
     'description' => 'TAO meta-extension',
     'license' => 'GPL-2.0',
-    'version' => '45.1.2',
+    'version' => '45.1.3',
```

Comment:
This looks like a new feature to me, kindly consider bumping a minor version instead.
```suggestion
    'version' => '45.2.0',
```